datadog-crashtracker/src/crash_info/builder.rs

```diff
@@ -364,7 +387,43 @@ impl CrashInfoBuilder {
         Ok(self)
     }
 
-    pub fn with_uuid_random(&mut self) -> anyhow::Result<&mut Self> {
-        self.with_uuid(Uuid::new_v4().to_string())
+    /// This method requires that the builder has a UUID, siginfo, and metadata set
+    pub fn build_crash_ping(&self) -> anyhow::Result<CrashPing> {
+        let uuid = self.uuid.clone().context("uuid is required")?;
+        let sig_info = self.sig_info.clone().context("sig_info is required")?;
+        let metadata = self.metadata.clone().context("metadata is required")?;
+
+        let builder = CrashPingBuilder::new()
+            .with_crash_uuid(uuid)
+            .with_sig_info(sig_info)
+            .with_metadata(metadata);
+        let crash_ping = builder.build()?;
```

Comment:
It looks like we're uploading a bunch of information even for a crash ping? what else is missing from crash ping compared to the actual crash log?